src/gen-harfbuzzcc.py

```diff
@@ -14,5 +14,8 @@
 with open (OUTPUT, "wb") as f:
 	f.write ("".join ('#include "{}"\n'.format (os.path.basename (x)) for x in sources if x.endswith (".cc")).encode ())
 
-# copy it also to src/
-shutil.copyfile (OUTPUT, os.path.join (CURRENT_SOURCE_DIR, os.path.basename (OUTPUT)))
+# copy it also to the source tree, but only if it has changed
+with open(os.path.join (CURRENT_SOURCE_DIR, os.path.basename (OUTPUT)), "r") as baseline:
```

Comment:
This long `os.path.join()` is repeated twice, can you make it a variable. Also the the `"r"` option is redundant, that is the default anyway.

Reply:
The filename is now a variable. I've kept the access mode and changed it to "rb" in line with the other script.